openpype/hosts/max/plugins/create/create_render.py

```diff
@@ -1,33 +1,28 @@
 # -*- coding: utf-8 -*-
 """Creator plugin for creating camera."""
 from openpype.hosts.max.api import plugin
-from openpype.pipeline import CreatedInstance
 from openpype.hosts.max.api.lib_rendersettings import RenderSettings
 
 
 class CreateRender(plugin.MaxCreator):
+    """Creator plugin for Renders."""
     identifier = "io.openpype.creators.max.render"
     label = "Render"
     family = "maxrender"
     icon = "gear"
 
     def create(self, subset_name, instance_data, pre_create_data):
-        from pymxs import runtime as rt
-        sel_obj = list(rt.selection)
-        instance = super(CreateRender, self).create(
+        """Plugin entry point."""
+        instance = super().create(
             subset_name,
             instance_data,
-            pre_create_data)  # type: CreatedInstance
+            pre_create_data)
         container_name = instance.data.get("instance_node")
-        container = rt.getNodeByName(container_name)
         # TODO: Disable "Add to Containers?" Panel
         # parent the selected cameras into the container
-        for obj in sel_obj:
-            obj.parent = container
-        # for additional work on the node:
-        # instance_node = rt.getNodeByName(instance.get("instance_node"))
-
-        # set viewport camera for rendering(mandatory for deadline)
-        RenderSettings().set_render_camera(sel_obj)
+        sel_obj = self.selected_nodes
+        if sel_obj:
+            # set viewport camera for rendering(mandatory for deadline)
+            RenderSettings().set_render_camera(sel_obj)
```

Comment:
Each creation of `RenderSettings` object will trigger get of settings. I would create only one object and pass settings (which are avaialble). Maybe it would worth to have the object in transient data?
```suggestion
            RenderSettings(self.project_settings).set_render_camera(sel_obj)
```